protocol/xmrmaker/swap_state_test.go

```diff
@@ -66,7 +62,7 @@ func newTestXMRMaker(t *testing.T) *Instance {
 		EthereumClient:       ec,
 		EthereumPrivateKey:   pk,
 		Environment:          common.Development,
-		ChainID:              big.NewInt(common.MainnetConfig.EthereumChainID),
```

Comment:
The current version of `ganache` is just called `ganache` instead of `ganache-cli`.  @noot assuming you agree, I'll create a PR that puts us on the current version of ganache in a subsequent PR after the multi-core issues are all solved. The newer versions of ganache throw an error when the chain ID is incorrect.

Reply:
sounds good!